bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx97-de.html

```diff
@@ -0,0 +1,76 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros-protocol.html" import card with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_97_eu') }}
+{% endblock %}
+
+{% set utm_campaign = campaign + '-de' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-nospace">
+      <div class="video-backdrop mzp-t-dark">
+        <div class="video-backdrop-inner">
+          <h1 class="wnp-main-title">Blicke mit uns auf das große Ganze</h1>
+          <p class="wnp-main-tagline">
+            Zusammen mit einer globalen Gemeinschaft von Machern und Denkern behalten wir das große Ganze im Auge. Um dir zu helfen, dich im Web zurechtzufinden und die Zukunft des Internets gemeinsam zu gestalten.
+          </p>
+
+          <p class="wnp-main-cta">
+            <a href="https://www.youtube.com/watch?v=_obdx3VUzuQ&cc_lang_pref=de&cc_load_policy=1" class="mzp-c-button mzp-t-product mzp-t-lg js-video-play">▶ Video abspielen</a>
```

Comment:
question (non-blocking): is it worth adding some GTM data attributes on this link to open the modal? (we already have the video-interaction data from the JS, so maybe not necessary here)